Directory.Build.props

```diff
@@ -9,6 +9,7 @@
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
+    <DisableImplicitNamespaceImports_DotNet>true</DisableImplicitNamespaceImports_DotNet>
```

Comment:
I haven't seen this issue and to my knowledge the PR/CI builds haven't had this issue either. I think you might be using a .NET 6 SDK version that is too old. I ask that you revert this change and try to update the SDK installed on your machine to the latest public release: https://dotnet.microsoft.com/download/dotnet/6.0

Reply:
I'll add a comment before marking this as resolved; my best guess is that it had to with being on an outdated version of the dogfood VS